src/control/client/connect.go

```diff
@@ -50,11 +50,14 @@ type Addresses []string
 
 // ConnFactory is an interface providing capability to create clients.
 type ConnFactory interface {
-	createConn(string) MgmtClient
+	createConn(string) Control
 }
 
 // Connections is an interface providing functionality across multiple
 // connected clients.
+//
+// TODO: find a way to not have to repeat client API here, maybe List(typeEnum)
```

Comment:
Not necessary to address in this review, but just a comment: In my view I think it's OK to have similar APIs here. You could rename to something like ListAllClientFeatures, ListAllClientNvme, etc. to emphasize the difference from the individual clients, but I'm not sure I understand what advantage we'd get from using generic List(), Provision(), etc. methods. Might be worth a wider discussion.